packages/core/data-transfer/lib/providers/local-strapi-source-provider/__tests__/index.test.ts

```diff
@@ -0,0 +1,112 @@
+import type { IEntity } from '../../../../types';
+
+import { Readable } from 'stream';
+
+import { collect, getStrapiFactory } from './test-utils';
+import { createLocalStrapiSourceProvider } from '../';
+
+describe('Local Strapi Source Provider', () => {
+  describe('Boostrap', () => {
+    test('Should not have a defined Strapi instance if bootstrap has not been called', () => {
+      const provider = createLocalStrapiSourceProvider({ getStrapi: getStrapiFactory() });
+
+      expect(provider.strapi).not.toBeDefined();
+    });
+
+    test('Should have a defined Strapi instance if bootstrap has been called', async () => {
+      const provider = createLocalStrapiSourceProvider({ getStrapi: getStrapiFactory() });
+      await provider.bootstrap();
+
+      expect(provider.strapi).toBeDefined();
+    });
+  });
+
+  describe('Close', () => {
+    test('Should destroy the strapi instance if autoDestroy is undefined ', async () => {
+      const destroy = jest.fn();
+
+      const provider = createLocalStrapiSourceProvider({
+        getStrapi: getStrapiFactory({ destroy }),
+      });
+
+      await provider.bootstrap();
+      await provider.close();
+
+      expect(destroy).toHaveBeenCalled();
+    });
+
+    test('Should destroy the strapi instance if autoDestroy is true ', async () => {
```

Comment:
You forgot "Should not destroy the strapi instance if autoDestroy is false" :)

Reply:
Nice, catch, fixed :slightly_smiling_face: